projects/auth0-angular/src/lib/auth.service.ts

```diff
@@ -161,11 +175,19 @@ export class AuthService<TAppState extends AppState = AppState>
     options?: PopupLoginOptions,
     config?: PopupConfigOptions
   ): Observable<void> {
-    return from(
-      this.auth0Client.loginWithPopup(options, config).then(() => {
-        this.authState.refresh();
-      })
-    );
+    const sub = new Subject<void>();
+    this.authClient
+      .getInstance$()
+      .pipe(
+        mergeMap((client) =>
+          client.loginWithPopup(options, config).then(() => {
+            this.authState.refresh();
+          })
+        )
+      )
+      .subscribe(sub);
+
+    return sub.asObservable();
```

Comment:
I have done this for backwards compatibility with the v1 API. I will add a follow up PR that cleans up this, but would like to dedicate a PR to it as it's a breaking change on its own.